.ci.yaml

```diff
@@ -78,7 +78,7 @@ platform_properties:
         ]
       os: Mac-12
       device_type: none
-      cpu: x86 # TODO(jmagman): https://github.com/flutter/flutter/issues/112130
+      cpu: arm64
```

Comment:
This line will be removed before merge.  I've temporarily set this to arm to force all "Mac" tests to run there to validate they pass on this PR.